
class OccupiedError(Exception):
    pass

class DeniedError(Exception):
    pass